docs/extensions/mb-blocks.md

```diff
@@ -160,6 +160,47 @@ By following WordPress standards, these parameters are available in the block te
 </div>
 ```
 
+#### Using MB Views
+
+You can also use MB Views to render the block. This is useful when you want to create views with twig, manage JS, CSS from within the Dashboard.
+
+To do so, first, make sure you have the [MB Views](/extensions/mb-views/) plugin installed and activated. Then, instead of using `file:` prefix, use `view:` prefix.
+
+```json
+{
+    "render": "view:testimonial"
+}
+```
+
+Now create a view by going to Meta Box > Views and create a new view with the name `testimonial`. In the view, you can use twig syntax to render the block. For example:
+
+```php
+<div {{ mb.get_block_wrapper_attributes() }}>
+	<img class="testimonial__image" src="{{ attributes['image']['full_url'] }}">
+	<div class="testimonial__body">
+		<div class="testimonial__content" style="min-height: 50px;">
+		    <InnerBlocks />
+		</div>
+		<div class="testimonial__author">{{ attributes['name'] }}</div>
```

Comment:
Chỗ này e update dùng dot trong attributes cho hay nhỉ, chứ dùng `[]` trông ko xịn. E giải thích thêm bằng lời nhé.

Reply:
Okie a, copy paste sang quên mất chỗ đó chứ e cũng toàn dùng dot.